astro.config.ts

```diff
@@ -10,12 +11,19 @@ import sitemap from "@astrojs/sitemap"
 // https://astro.build/config
 export default defineConfig({
   site: "https://docs.chain.link",
-  // trailingSlash: 'never',
-  integrations: [preact(), react(), sitemap({ changefreq: "daily" }), mdx()],
+  integrations: [
+    preact({
+      include: ["**/preact/*"],
```

Comment:
Question: isn't this indicating that we are having the preact components in a preact folder? Since the components are not currently organized this way, is this doing anything currently?

Reply:
Specifying `include` is necessary for the integrations to work properly on 3.0, actually having them in the folders is optional